video-app/src/app/core/models/platforms/vimeo/vimeo.response.ts

```diff
@@ -0,0 +1,10 @@
+import { VimeoPictures } from './vimeoPictures';

```

Comment:
jeżeli już chcesz robić dla tego osobny interfejs, a nie trzymać go bezpośrednio w `VimeoResponse` to dodaj go w tym pliku bo za dużo tych plików się robi przy samym interfejsach yt/vimeo